deepcell/applications/application_test.py

```diff
@@ -34,6 +34,7 @@
 from tensorflow.keras.layers import Input
 from tensorflow.python.platform import test
 from unittest.mock import Mock
+from itertools import product
```

Comment:
This should actually go in its own block, between the future imports and the 3rd party imports. (same with `unittest`):

```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

from itertools import product
from unittest.mock import Mock

import numpy as np
from tensorflow.keras.layers import Input
from tensorflow.python.platform import test
```